test/e2e-tests.sh

```diff
@@ -40,6 +40,12 @@
 #   PROJECT_ID:  the GKR project in which to create the new cluster (unless using "--run-tests")
 #   MAX_PARALLEL_TESTS:  The maximum number of go tests to run in parallel (via "-test.parallel", default 12)
 
+# FIXME(ksuszyns): Remove upgrade-tests invocation before merging the PR
```

Comment:
@cardil what about this `FIXME` ?

Reply:
As it said. I'll remove it after review. This temporarily runs upgrade tests before https://github.com/knative/test-infra/pull/2735 is merged

Reply:
OK